engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/metrics/MetricsResultDto.ftl

```diff
@@ -0,0 +1,9 @@
+<@lib.dto>
+
+    <@lib.property
+        name = "result"
+        type = "number"
```

Comment:
From what I can see from the (only) usage, this is a Java `long` (see [here](https://github.com/camunda/camunda-bpm-platform/blob/6fd62f5c7492b98f0187f0392dc1ea2a94342cda/engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/sub/metrics/MetricsResourceImpl.java#L54)). The OpenAPI spec defines the `number` type as `float/double`. So we need to set this to:
```suggestion
        type = "integer"
        format = "int64"
```